installer/bundle/bundle_skel.sh

```diff
@@ -1456,10 +1456,8 @@ case "$installMode" in
                 OSS_BUNDLE=`basename $i -oss-test.sh`
                 [ ! -f oss-kits/${OSS_BUNDLE}-cimprov-*.sh ] && continue
 
-                if [ "$OSS_BUNDLE" = "docker" ]  && [ "$skipDockerProviderInstall" = "true" ]; then
-                   echo "$OSS_BUNDLE provider package installation skipped since --skip-docker-provider-install flag is set. \
-If you are installing over an existing omsagent install and wish to remove the docker provider, \
-you should first purge the existing installation and then install using the --skip-docker-provider-install flag."
+                # New installations of the docker-cimprov package are no longer supported
```

Comment:
since the option --skip-docker-provider-install being ignored for the new installations with this change, we should remove this option for supported install command options